.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: CI
+
+on:
+  - pull_request
+  - push
+
+env:
+  TESTOPT: "-v"
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Ruby
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.0"
+          - "2.1"
+          - "2.2"
+          - "2.3"
+          - "2.4"
+          - "2.5"
+          - "2.6"
+          - "2.7"
+          - "3.0"
+          - "3.1"
+          - "ruby-head"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: bundle exec rake
```

Comment:
Is there a reason to use Bundler here?

Reply:
Mostly because it's what I'm used to. Another benefit is that the GitHub action creates a cache for the next CI run. It integrates nicely with the `Gemfile`. Note I had to add `rake` there since it wasn't present in Ruby >= 2.3 (IIRC, could be 2.4).

Reply:
I think that Bundler was always used for installation, but never for execution. Using Bundler has possibly some side effects and I'd like to avoid it.

Also, as far as I can tell, Rake was first included as a bundled gem in Ruby 2.3 [1], but prior that, it was part of StdLib. Therefore I think that if Bundler is not used for execution, we should be fine without it being specified in Gemfile (and these are the side effect of Bundler and why I don't think Bundler should be used for test suite execution).


[1]: https://github.com/ruby/ruby/commit/c4ee0df8ba2cf7cc6aaa785c8df20a91b1719021

Reply:
Ok, I guess it was just an odd issue because when I reran it it passed: https://github.com/ekohl/gem2rpm/actions/runs/2628526691